src/nidcpower/metadata/attributes.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-DCPower API metadata version 22.8.0d88
```

Comment:
await official metadata build

Reply:
I have updated the metadata version to 22.8.0d129 in my latest commit